static/sass/styles.scss

```diff
@@ -1766,3 +1766,11 @@ legend {
     height: 15rem;
   }
 }
+
+// XXX: Custom styling to remove the full-stop after the counter in 'p-stepped-list'
```

Comment:
Could you create an issue on vanilla to purpose this change please?

Reply:
PR created [here](https://github.com/canonical/vanilla-framework/pull/5465)